src/main/java/com/assignment/application/repo/CompanyRepo.java

```diff
@@ -20,7 +19,6 @@
             " emp.project_id as project_id,emp.phone_number as phone_number,emp.current_add as current_add,emp.permanent_add as permanent_add from Employee emp" +
```

Comment:
Use DSL wherever required.

Reply:
Try to do it with join column and fetchmode or may be implement a custom repo with criteria query with hibernate